torch/_dynamo/variables/user_defined.py

```diff
@@ -727,6 +727,10 @@ class NO_SUCH_SUBOBJ:
                 unimplemented("UserDefined with non-function __getattr__")
 
         if isinstance(subobj, property):
+            # Rewrite the source being explicit about reading it statically.
+            if self.source:
+                source = AttrSource(self.source, name, get_static=True)
+                source = AttrSource(source, "fget")
```

Comment:
curious, what's this fget here?

Reply:
Python property classes are made of three parts:
fget - the getter function
fset - the setter function
fdel - the `del` function

So if we're reading the value on the property we want to access the underlying 'fget' function. This is what the code below here was already doing (`subobj.fget`) but we weren't registering that fact with the guard.